models/workitem_repository_blackbox_test.go

```diff
@@ -0,0 +1,80 @@
+package models_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/criteria"
+	"github.com/almighty/almighty-core/models"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateLoadDeleteListWI(t *testing.T) {
+	doWithTransaction(t, func(ts *models.GormTransactionSupport) {
```

Comment:
How many things are we testing at once? Wouldn't it be better to split it to more focused, descriptive tests? This test itself has too many reasons to fail in my opinion.